### ISSUE TITLE
WIKICATEGORYMEMBERS does not support non-mainspace articles

### ISSUE THREAD

**Issue by Sadads** [tomayac/wikipedia-tools-for-google-spreadsheets#1]
Hey, I love the tool, but really want to be able to create a list of "Wikipedia:" namespace pages, with WIKICATEGORYMEMBERS and the other linking tools. For example, the current is not returning outcomes for https://en.wikipedia.org/wiki/Category:The_Wikipedia_Library . A lot of programs and community activity regularly use other namespaces, such as User pages, talk pages, and Wikipedia: space (or project space). For those of us who do evaluation and assessment of the community, and would be really valuable, to be able to pull that data quickly, and without having to work through other tools, quarry or the API proper. Having an additional variable that allowed a call on different namespaces, would be incredibly powerful.

**Comment by Sadads** [comment]
Hey @tomayac have you had a chance to look at this?

**Comment by tomayac** [comment]
Not yet, but I have not forgotten about it, no worries. What would help in order to have a holistic solution (and not just fix one particular function) would be if you could list all "namespace-able" functions here. Thanks.

**Comment by Sadads** [comment]
Sure:
* WIKICATEGORYMEMBERS per above
*Not sure how WIKIARTICLESAROUND is calculated, but it might be worth limiting to namespaces (Say I wanted to figure out the relationships between policies on a particular wiki in the Wikipedia: Space, or I was interested in categories or templates). 
*WIKIINBOUNDLINKS, WIKIMUTUALLINKS, WIKIOUTBOUNDLINKS (specially, being able to distinguish between mainspace, and other kinds of spaces -- you may want to know if the article has been talked on in talk pages for instance, or is referenced in Wikipedia: space)
*WIKISEARCH -- probably can piggy back on MediaWiki's search by namespace features already there. For this one, you probably want a field that allows for multiple namespaces.

I think, of the existing tools, those are the only ones where you don't specifically query a title, so it would be worth being able to limit the outcomes.

**Comment by tomayac** [comment]
Sorry that this took longer than expected, but I have finally committed https://github.com/tomayac/wikipedia-tools-for-google-spreadsheets/commit/b8139fe3c04904237339ff3369790ab9b0e66ced that closes https://github.com/tomayac/wikipedia-tools-for-google-spreadsheets/issues/1. It has just been released as version 13 on the Google Sheets Add-on Store (you might need to reload Google Sheets for it to be updated). @Sadads, hope this works fine for you :-) You pass the namespaces as pipe-separated values like so: ```0|1|2```. Thanks for your patience!
